packages/ocamline/ocamline.1.1/opam

```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+synopsis: "Command line interface for user input"
+maintainer: "Chris Nevers <christophernevers96@gmail.com>"
+authors: "Chris Nevers <christophernevers96@gmail.com>"
+homepage: "https://github.com/chrisnevers/ocamline"
+bug-reports: "https://github.com/chrisnevers/ocamline/issues"
+depends: [
```

Comment:
```suggestion
depends: [
  "ocaml"
```
I'd also suggest to be explicit about the dependency on the ocaml compiler. This makes it easier to patch the package later